src/Resources/Reusables/DisplayCard/DisplayCard.jsx

```diff
@@ -1,23 +1,49 @@
-import { Card } from 'react-bootstrap';
+//Import from react-bootstrap 
+import { Button, Card } from 'react-bootstrap';
+
+//Import style
 import './DisplayCard.css';
-import testImage from '../../../Assets/Charity.jpeg'
-
-function DisplayCard({name, phone, phoneNumber, image, details}) {
-    return(
-        <Card style={{}}>
-            <Card.Img variant="top" src={image ? image.url : testImage} />
-            <Card.Body>
-              <Card.Title>{name}</Card.Title>
-              <Card.Subtitle>Phone Number</Card.Subtitle>
-              <br/>
-              <Card.Subtitle>{phone}</Card.Subtitle>
-              <Card.Text>
-                {phoneNumber}
-              </Card.Text>
-              <Card.Link href='/home'>More details</Card.Link>
-            </Card.Body>
-        </Card>
-    );
+
+//Import default image
+import testImage from '../../../Assets/SalamLogo.jpeg';
```

Comment:
If test Image is the "fallback image or alt image can we rename it to be clearer?

Reply:
Thought I changed it.